test/src/evented-queue.spec.js

```diff
@@ -0,0 +1,63 @@
+const Subject = require('../../src/evented-queue');
+
+describe('src/evented-queue', () => {
+	let instance;
+
+	beforeEach(() => {
+		instance = new Subject();
+	});
+
+	describe('#add', () => {
+		it('adds the given item to the queue', () => {
+			instance.add('foo');
+			expect(instance.queue.size).toEqual(1);
+		});
+
+		it('emits an event when items are added to the queue', (done) => {
```

Comment:
should this test have an 'expect' declaration?

Reply:
Good question! But no it shouldn't. The `done()` callback in this case is the success criteria, failure would be not calling it.